biz.aQute.bnd.reporter/test/biz/aQute/bnd/reporter/codesnippet/CodeSnippetExtractorTest.java

```diff
@@ -0,0 +1,34 @@
+package biz.aQute.bnd.reporter.codesnippet;
+
+import biz.aQute.bnd.reporter.codesnippet.dto.CodeSnippetDTO;
+import biz.aQute.bnd.reporter.codesnippet.dto.CodeSnippetGroupDTO;
+import biz.aQute.bnd.reporter.codesnippet.dto.CodeSnippetProgramDTO;
+import java.io.File;
+import java.util.List;
+import junit.framework.TestCase;
+
+public class CodeSnippetExtractorTest extends TestCase {
```

Comment:
I think we need a test cases added to the reporter maven plugin as well.

Reply:
I have added junit tests for the maven plugins.